packages/components/src/styling/src/styled-system/props-handler.ts

```diff
@@ -0,0 +1,1672 @@
+    "neutral-weak-hover": "neutral-surface-weak-hover",
+    "neutral-hover": "neutral-surface-hover",
+    "neutral-strong": "neutral-surface-strong",
+    "neutral-weak": "neutral-surface-weak",
+    "neutral-active": "neutral-surface-active",
+    "neutral-disabled": "neutral-surface-disabled",
+    "neutral-weakest-hover": "neutral-surface-weakest-hover",
+    "neutral-weakest-active": "neutral-surface-weakest-active",
+    "decorative-option1-hover": "decorative-option1-surface-hover",
+    "decorative-option1-strong": "decorative-option1-surface-strong",
+    "decorative-option1-weakest": "decorative-option1-surface-weakest",
+    "decorative-option1-weak": "decorative-option1-surface-weak",
+    "decorative-option2-hover": "decorative-option2-surface-hover",
+    "decorative-option2-weak": "decorative-option2-surface-weak",
+    "decorative-option3-hover": "decorative-option3-surface-hover",
+    "decorative-option3-strong": "decorative-option3-surface-strong",
+    "decorative-option3-weakest": "decorative-option3-surface-weakest",
+    "decorative-option3-weak": "decorative-option3-surface-weak",
+    "decorative-option4-hover": "decorative-option4-surface-hover",
+    "decorative-option4-strong": "decorative-option4-surface-strong",
+    "decorative-option6-strong": "decorative-option6-surface-strong",
+    "decorative-option6-hover": "decorative-option6-surface-hover",
+    "decorative-option6-weak": "decorative-option6-surface-weak",
+    "decorative-option5-hover": "decorative-option5-surface-hover",
+    "decorative-option5-strong": "decorative-option5-surface-strong",
+    "decorative-option7-hover": "decorative-option7-surface-hover",
+    "decorative-option7-strong": "decorative-option7-surface-strong",
+    "decorative-option8-strong": "decorative-option8-surface-strong",
+    "decorative-option8-hover": "decorative-option8-surface-hover",
+    "decorative-option9-hover": "decorative-option9-surface-hover",
+    "decorative-option9-strong": "decorative-option9-surface-strong",
+    "primary-active": "primary-surface-active",
+    "primary-hover": "primary-surface-hover",
+    "primary-strong-hover": "primary-surface-strong-hover",
+    "primary-strong-active": "primary-surface-strong-active",
+    "primary-strong": "primary-surface-strong",
+    "primary-weak": "primary-surface-weak",
+    "primary-focus": "primary-surface-focus",
+    "primary-disabled": "primary-surface-disabled",
+    "danger-active": "danger-surface-active",
+    "danger-weak": "danger-surface-weak",
+    "danger-strong-hover": "danger-surface-strong-hover",
+    "danger-strong": "danger-surface-strong",
+    "danger-hover": "danger-surface-hover",
+    "danger-disabled": "danger-surface-disabled",
+    "upsell-active": "upsell-surface-active",
+    "upsell-disabled": "upsell-surface-disabled",
+    "upsell-weak": "upsell-surface-weak",
+    "upsell-hover": "upsell-surface-hover"
+} as const;
+
+const BorderWidthAndStyle = "1px solid";
```

Comment:
Should this be a rem?

Reply:
I think it's one of the few usecase where i'd want an hardcoded border? i never want the border to be less than a pixel